### ISSUE TITLE
Naive question about tutorial

### ISSUE THREAD

**Issue by gouthamatla** [FunctionLab/selene#78]
Hi,

Thanks for developing this package. Its very useful for people who wants to get started with machine learning in genomics.

I have a very naive question about Getting Started tutorial.  I would like to know whats the specific role of "deepsea_TF_intervals.txt" file in training the model. Are you using the deepsea_TF_intervals file to predict the CTCF binding sites ? or the model is trained on certain number of CTCF peaks and tries to predict the rest ? in this case whats the role of deepsea_TF_intervals ?

Thanks,
Goutham A 

PS: Is there any google groups to ask some methodological or theoretical questions ?

**Comment by kathyxchen** [comment]
Hi Goutham,

Thanks for your question! The intervals file is used to specify the intervals from which Selene draws samples. In this case, we are restricting the regions to sample from to the regions that contain at least one transcription factor (any TF, not just CTCF)--those regions will contain other chromatin features, so at the end of training the model should still "see" all chromatin features that it wants to predict. The restriction of intervals in this situation just speeds up the time it takes to train for this particular dataset (compared to sampling from the whole genome, which would cause the model to encounter more regions with only histones/very few chromatin features). 

Hope that clears things up a bit - thank you for trying out the tutorial! Let us know if you have more questions/comments (if unrelated to the getting started tutorial, please feel free to post as a separate issue).

We don't currently have a google group, but I'll look into whether it makes sense to set one up! Thanks for the suggestion.

**Comment by gouthamatla** [comment]
Thanks for the explanation. I am just getting started with ML. 

At the begging of the tutorial, from point 2 "2. Load the training data and divide it into training and validation sets." Presumably "**distinct_features.txt**" is split into training and validation set and **deepsea_TF_intervals.txt** is a test set ? Is my understanding correct ?

**Comment by kathyxchen** [comment]
No, `deepsea_TF_intervals.txt` is the regions from which we are allowed to sample. `sorted_GM12878_CTCF.bed.gz` should be split into training, validation, and testing sets. Could you link to the part in the tutorial description that you found confusing, in case we need to adjust the language there? 

The key point to know is that `sorted_GM12878_CTCF.bed.gz` is a compressed BED file of all the "positive samples" that Selene will generate for training. That is, this is the list of regions for which we have the CTCF measurements. `deepsea_TF_intervals.txt` does not contain any information about whether a CTCF peak was present - it is only a list of regions from which we would like to draw positive or negative samples. The reason it is named `deepsea_TF_intervals` is because this is the list of regions sampled from in the original DeepSEA paper. I realize that the `TF_intervals` part is confusing, sorry about that!

As for the Google groups question you had in the initial post, we do not plan to create one at this time. Unfortunately, we do not have the bandwidth to support it currently, but maybe in the future.

(Apologies for the delayed response - I was out of office last week.)

**Comment by evancofer** [comment]
@kathyxchen @gouthamatla Can this be closed?

**Comment by kathyxchen** [comment]
can re-open if needed
